scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -1,6 +1,6 @@
 version 1.0
 
-import "AlignmentPipeline.wdl" as AlignAndMarkDuplicates
+import "https://api.firecloud.org/ga4gh/v1/tools/mitochondria:AlignmentPipeline/versions/1/plain-WDL/descriptor" as AlignAndMarkDuplicates
```

Comment:
I think for the tests to work this needs to be the file name "AlignmentPipeline.wdl". If there's a better solution for moving things back and forth between Terra and the gatk repo though I'm all for it.